src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+@import './variables';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
 body {
   margin: 0;
 }
+
+h3 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    align-self: center;
+  }
+
+  &__description {
+    font-size: inherit;
+    font-weight: inherit;
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
```

Comment:
The default value, so you can just remove it everywhere
```suggestion
```

Reply:
@volodymyr-soltys97 the default values for my font described in html tag at the start of css file, and my default value for font-weight is 500
![image](https://github.com/mate-academy/layout_product-cards/assets/156140967/e32c737d-4504-4c2e-bbba-33b66995fac6)